test/valid-data/type-pick/main.ts

```diff
@@ -0,0 +1,8 @@
+
+export interface MyInterface {
+    numberField: number,
+    stringField: string
+    string2Field: {}
+}
+
+export interface MyObject extends Pick<MyInterface, "numberField"|"stringField"> {}
```

Comment:
add space around |

Reply:
RE getting it to work with vega lite, you're probably best positioned to create a test case for that, as you're more familiar with the vega lite code base.

Reply:
I think if you create a test case with a pick with an interface, you will see the issue.